### PR TITLE
Fixed cosmic resampling

### PR DESCRIPTION
**Pull request by soleti**
This PR adds the `compressPV` producer to the stage-1 cosmic production and the `volumeInfoMixer` to the stage-2 DS resampling.

### CODE REVIEW

File: JobConfig/cosmic/S1DSStops.fcl [hunk 1]
```diff
@@ -6,8 +6,8 @@
 
 process_name : Primary
 
-physics.TriggerPath: [generate, genCounter, g4run, g4status, stepPointMomentumFilter, cosmicFilter]
-outputs.Output.outputCommands : [@sequence::Primary.KeptProducts, "keep mu2e::GenParticles_*_*_*", "keep *_cosmicFilter_*_*"]
+physics.TriggerPath: [generate, genCounter, g4run, g4status, stepPointMomentumFilter, cosmicFilter, compressPV]
```
Comment (brownd1978) [inline JobConfig/cosmic/S1DSStops.fcl:9]: I don't see where the compressPV module is defined.  It's not in this file or any of the #includes.

Comment (soleti) [reply JobConfig/cosmic/S1DSStops.fcl:9]: It's in JobConfig/primary/prolog.fcl (but defined in JobConfig/cosmic/prolog.fcl) no?

**Comment by brownd1978** [comment]
I must be blind as I can't see it in JobConfig/primary/prolog.fcl  Can you please point me to the line?

File: JobConfig/primary/prolog.fcl [hunk 2]
```diff
@@ -22,6 +22,7 @@ Primary: {
     @table::TrackerMC.StepProducers
     @table::CaloMC.StepProducers
     CrvSteps : @local::CrvSteps
+    compressPV: @local::Cosmic.compressPV
```
Comment (soleti) [inline JobConfig/primary/prolog.fcl:25]: @brownd1978 here

Comment (brownd1978) [inline JobConfig/primary/prolog.fcl:25]: I see it's something you added.  primary/prolog should not instantiate a module from Cosmic, that's a reverse-dependency.  Please move this to cosmic/prolog.fcl

File: JobConfig/cosmic/prolog.fcl [hunk 3]
```diff
@@ -103,15 +109,19 @@ Cosmic: {
    }
    dsResample: {
```
Comment (brownd1978) [inline JobConfig/cosmic/prolog.fcl:110]: Why does a cosmic job have a module named 'dsResample'?  Was this just a copy from Andrei's example?  I think a name like cosmicResampler would be more intuitive.

**Review by brownd1978 — APPROVED** [review]
looks good

**Comment by brownd1978** [comment]
@FNALbuild run build test

**Review by oksuzian — APPROVED** [review]
Hi Roberto,
I believe we wanted to structure our fcl file starting with include of "JobConfig/primary/primary.fcl". Other than that I'm fine with this PR

**Comment by brownd1978** [comment]
S2 jobs should inherit from primary.fcl, but S1 jobs are a different type and should not inherit from primary.fcl

**Pull request merged**